src/webui/src/components/PackageSidebar/modules/LastSync/index.jsx

```diff
@@ -19,14 +19,18 @@ export default class LastSync extends React.Component {
       }
     });
 
-    return lastUpdate ? datetime(lastUpdate) : '';
+    const datetime = format(new Date(lastUpdate), 'MMM D, YYYY, h:mm A');
```

Comment:
Create a common constant of this format in a common place `'MMM D, YYYY, h:mm A'`

Reply:
I don't think this is a good idea for a international project. In my country, we use `YYYY/MM/DD HH:mm:ss` not `MMM D, YYYY, h:mm A`, it's really hard to read for some people

Reply:
What could we use?

Reply:
@juanpicado Follow browser or system settings
```js
(new Date(timestamp).toLocaleString())
```

Reply:
But test might be run on a different system, we need to avoid side effects on a test environment.

Reply:
@Meeeeow the current implementation replaces the (new Date(timestamp).toLocaleString()) in different os environments.